### PR TITLE
Multiple code improvements - squid:S1118, squid:S1192

### PR DESCRIPTION
**Pull request by georgekankava**
This pull request is focused on resolving occurrences of Sonar rules
squid:S1118 - Utility classes should not have public constructors.
squid:S1192 - String literals should not be duplicated.
You can find more information about the issues here:
https://dev.eclipse.org/sonar/rules/show/squid:S1118
https://dev.eclipse.org/sonar/rules/show/squid:S1192
Please let me know if you have any questions.
George Kankava

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@georgekankava, it looks like you've already signed the Dropbox CLA. Thanks!

File: app/src/main/java/com/zulip/android/AsyncGetEvents.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
 import com.j256.ormlite.misc.TransactionManager;
 
 public class AsyncGetEvents extends Thread {
+    public static final String ASYNC_GET_EVENTS = "asyncGetEvents";
+    public static final String POINTER = "pointer";
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/AsyncGetEvents.java]: If these are only used internally to this class, it's a good practice to make their visibility `private`.

File: app/src/main/java/com/zulip/android/AsyncStatusUpdate.java [hunk 2]
```diff
@@ -18,6 +18,9 @@
  */
 public class AsyncStatusUpdate extends ZulipAsyncPushTask {
 
+    public static final String STATUS = "status";
+    public static final String TIMESTAMP = "timestamp";
+    public static final String STATUS_UPDATE = "statusUpdate";
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/AsyncStatusUpdate.java]: see above comment.

File: app/src/main/java/com/zulip/android/CustomHtmlToSpannedConverter.java [hunk 3]
```diff
@@ -62,6 +62,7 @@
 
     private static final float[] HEADER_SIZES = {1.5f, 1.4f, 1.3f, 1.2f, 1.1f,
             1f,};
+    public static final String MONOSPACE = "monospace";
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/CustomHtmlToSpannedConverter.java]: same as above.

File: app/src/main/java/com/zulip/android/ZulipActivity.java [hunk 4]
```diff
@@ -53,6 +53,8 @@
 public class ZulipActivity extends FragmentActivity implements
         MessageListFragment.Listener {
 
+    public static final String NARROW = "narrow";
+    public static final String PARAMS = "params";
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/ZulipActivity.java]: see above comment.

**Comment by niftynei** [comment]
Hi, thanks for the PR.  These changes look really good. :+1: 

For clarity's sake, it'd be helpful if you could split each of the code changes into commits corresponding with each of the rules that they're fixing.

The Zulip version control doc is a good reference for this: https://zulip.readthedocs.org/en/latest/code-style.html#version-control

**Comment by georgekankava** [comment]
I will resubmit this as separate pull requests.

**Pull request closed without merging**